setup.py

```diff
@@ -1,11 +1,11 @@
 import setuptools, os
 
-PACKAGE_NAME = 'facenet-pytorch'
-VERSION = '2.5.2'
-AUTHOR = 'Tim Esler'
-EMAIL = 'tim.esler@gmail.com'
-DESCRIPTION = 'Pretrained Pytorch face detection and recognition models'
-GITHUB_URL = 'https://github.com/timesler/facenet-pytorch'
+PACKAGE_NAME = 'facenet-pytorch-custom'
+VERSION = '2.5.2.dev2'
+AUTHOR = 'Shivam Singhal'
```

Comment:
Was this intentionally added to the PR? I can't merge a change that changes the package author